src/main/yamcs/etc/yamcs.oresat0.yaml

```diff
@@ -59,6 +62,22 @@ dataLinks:
     port: 10016
     packetPreprocessorClassName: org.oresat.uniclogs.tctm.EdlPacketPreprocessor
 
+  - name: satnogs-tc-realtime
+    class: org.yamcs.tctm.UdpTcDataLink
+    stream: satnogs_tc_realtime
+    host: localhost
+    port: 10017
+    commandPostprocessorClassName: org.oresat.uniclogs.tctm.SatnogsCommandPostprocessor
```

Comment:
Addressed in other comments, but this becomes an extraneous post-processor once all the CMD components are axed.